spec/integration/twingly/amqp/utilities_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe Twingly::AMQP::Utilities do
+  let(:queue_name)      { "twingly-amqp.utilities.test.queue" }
+  let(:amqp_connection) { Twingly::AMQP::Connection.instance }
+
+  subject do
+    described_class.create_queue(queue_name, connection: amqp_connection)
+  end
+
+  after do
+    amqp_connection.with_channel do |channel|
+      channel.queue_delete(queue_name)
+    end
+  end
+
+  it "creates a queue" do
```

Comment:
```suggestion
  it "creates the queue" do
```

a bit more strict, "a" sounds like we would accept any queue with any name

/nitpicking